src/main/java/racingCar/Car.java

```diff
@@ -0,0 +1,29 @@
+package racingCar;
+
+import java.util.Random;
+
+public class Car {
+    private static final Random random = new Random();
+    private final int MOVE_MIN_NUM = 4;
+    private final int name;
```

Comment:
상수와 인스턴스 변수의 구분을 위해 공백 라인을 추가해보면 좋을 것 같아요.
```suggestion
    private final int MOVE_MIN_NUM = 4;
    
    private final int name;
```